modules/ref_ipi-installation-overview.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/ipi/ipi-install-prerequisites.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="installation-overview_{context}"]
+= Installation overview
+
+The installation program supports interactive mode, but Red{nbsp}Hat recommends preparing an `install-config.yaml` file in advance, containing the provisioning details for all of the bare-metal hosts, and the relevant cluster details.
```

Comment:
Per the SSG: [Avoid "recommends". Instead of "Red Hat recommends", direct users to take the recommended action](https://redhat-documentation.github.io/supplementary-style-guide/#recommend). Also, perhaps move the _in advance_ for better flow?
Maybe:

```suggestion
The installation program supports interactive mode. However, you can prepare an `install-config.yaml` file that contains the provisioning details for all of the bare-metal hosts, and the relevant cluster details, in advance.
```